src/Apps/Artwork/Components/__stories__/Banner.story.tsx

```diff
@@ -1,30 +1,31 @@
 import React from "react"
 import { storiesOf } from "storybook/storiesOf"
 import { Section } from "Styleguide/Utils/Section"
-import { Banner, LargeBanner, SmallBanner } from "../Banner"
+import { Banner, LargeBanner, SmallBanner } from "../ArtworkBanner/Banner"
 
 storiesOf("Styleguide/Artwork", module).add("Banner", () => {
   return (
     <React.Fragment>
-      <Section title="Responsive">
+      <Section title="Responsive with image">
         <Banner
-          src="https://picsum.photos/110/110/?random"
+          imageUrl="https://picsum.photos/110/110/?random"
           badge="In show"
           headline="Francesca DiMattio: Boucherouite"
           subHeadline="Salon 94"
         />
       </Section>
-      <Section title="Large Banner">
+      <Section title="Large Banner with initials">
         <LargeBanner
-          src="https://picsum.photos/110/110/?random"
+          imageUrl={null}
+          initials="OU"
           badge="In show"
           headline="Francesca DiMattio: Boucherouite"
           subHeadline="Salon 94"
         />
       </Section>
       <Section title="Small Banner">
         <SmallBanner
-          src="https://picsum.photos/110/110/?random"
+          imageUrl="https://picsum.photos/110/110/?random"
```

Comment:
I think this file can go away once we implement `EntityHeader` and instead rely on `ArtworkBanner`

Reply:
@damassi - I left it here as a test for responsive versions of the Banner. The actual data for the Artwork Banner is storied in ArtworkBanner file.